From 63a2009bb43d7d5d466f1bb9327e013166f8b022 Mon Sep 17 00:00:00 2001
From: "xiaoya.lin" <xiaoya.lin@amlogic.com>
Date: Mon, 24 Jul 2023 11:05:02 +0800
Subject: [PATCH 2/2] gstamlvenc: Fixed log err [1/1]

PD#SWPL-132055

Problem:
Fixed log err

Solution:
Fixed log err

Verify:
Local

Signed-off-by: xiaoya.lin <xiaoya.lin@amlogic.com>
Change-Id: I16d07085dca1a04d7d91fcce51ec0afc7e417b7f
---
 amlvenc/gstamlvenc.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/amlvenc/gstamlvenc.c b/amlvenc/gstamlvenc.c
index 5308d10..c191c57 100644
--- a/amlvenc/gstamlvenc.c
+++ b/amlvenc/gstamlvenc.c
@@ -222,8 +222,8 @@ static amvenc_img_format_t img_format_convert (GstVideoFormat vfmt)
             break;
         case GST_VIDEO_FORMAT_I420:
         case GST_VIDEO_FORMAT_YV12:
-                fmt = AML_IMG_FMT_YUV420P;
-                break;
+            fmt = AML_IMG_FMT_YUV420P;
+            break;
         case GST_VIDEO_FORMAT_RGB:
             fmt = AML_IMG_FMT_RGB888;
             break;
@@ -1002,7 +1002,7 @@ static GstFlowReturn gst_amlvenc_encode_frame (GstAmlVEnc * encoder, GstVideoCod
             inbuf_info.buf_info.dma_info.shared_fd[0] = encoder->amlge2d.ge2dinfo.dst_info.shared_fd[0];
             inbuf_info.buf_info.dma_info.num_planes = ui1_plane_num;
             encoder->amlge2d.ge2dinfo.src_info[0].shared_fd[0] = -1;
-            GST_DEBUG_OBJECT(encoder, "Set DMA buffer index %d planes %d fd[%d]",
+            GST_DEBUG_OBJECT(encoder, "Set DMA buffer planes %d fd[%d]",
             inbuf_info.buf_info.dma_info.num_planes, inbuf_info.buf_info.dma_info.shared_fd[0]);
         } else
 #endif
@@ -1012,7 +1012,7 @@ static GstFlowReturn gst_amlvenc_encode_frame (GstAmlVEnc * encoder, GstVideoCod
             inbuf_info.buf_info.dma_info.shared_fd[1] = encoder->fd[1];
             inbuf_info.buf_info.dma_info.shared_fd[2] = encoder->fd[2];
             inbuf_info.buf_info.dma_info.num_planes = ui1_plane_num;
-            GST_DEBUG_OBJECT(encoder, "Set DMA buffer index %d planes %d fd[%d, %d, %d]",
+            GST_DEBUG_OBJECT(encoder, "Set DMA buffer planes %d fd[%d, %d, %d]",
             inbuf_info.buf_info.dma_info.num_planes, inbuf_info.buf_info.dma_info.shared_fd[0],
             inbuf_info.buf_info.dma_info.shared_fd[1], inbuf_info.buf_info.dma_info.shared_fd[2]);
         }
@@ -1051,7 +1051,7 @@ static GstFlowReturn gst_amlvenc_encode_frame (GstAmlVEnc * encoder, GstVideoCod
             inbuf_info.buf_type = AML_DMA_TYPE;
             inbuf_info.buf_info.dma_info.shared_fd[0] = encoder->amlge2d.ge2dinfo.dst_info.shared_fd[0];
             inbuf_info.buf_info.dma_info.num_planes = ui1_plane_num;
-            GST_DEBUG_OBJECT(encoder, "Set DMA buffer index %d planes %d fd[%d]",
+            GST_DEBUG_OBJECT(encoder, "Set DMA buffer planes %d fd[%d]",
             inbuf_info.buf_info.dma_info.num_planes, inbuf_info.buf_info.dma_info.shared_fd[0]);
             gst_video_frame_unmap (&video_frame);
         } else
-- 
2.25.1

